notebooks/303-pytorch-int8-semseg-with-nncf-pot-openvino/303-pytorch-int8-semseg-with-nncf-pot-openvino.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "af7b6f13",
+   "metadata": {},
+   "source": [
+    "# INT8 acceleration of semantic segmentation with NNCF/POT for OpenVINO\n",
+    "\n",
+    "This notebook runs through the process of enabling [NNCF](https://github.com/openvinotoolkit/nncf) in a simple semantic segmentation pipeline for training UNet on Camvid, and comparing results against the Post-Training Optimization Tool ([POT](https://docs.openvinotoolkit.org/latest/pot_README.html))'s quantization.\n",
+    "\n",
+    "> NOTE: _For this notebook to function, please make sure that your Python environment has `openvino`, `openvino-dev` and `nncf[torch]` packages installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "27cb5c08",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\npython = sys.executable\n%ls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "49c29922",
+   "metadata": {},
+   "source": [
+    "Get the dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2a33c1af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/alexgkendall/SegNet-Tutorial data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "24d32df7",
+   "metadata": {},
+   "source": [
+    "### Obtaining the uncompressed (FP32) performance and accuracy baselines\n",
+    "Download the FP32 pre-trained weights checkpoint for UNet on CamVid:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4ad25d7b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!curl -o unet_camvid.pth https://storage.openvinotoolkit.org/repositories/nncf/openvino_notebook_ckpts/303_unet_camvid.pth"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4a2f0f0",
+   "metadata": {},
+   "source": [
+    "Measure the baseline FP32 accuracy in PyTorch and produce an ONNX for future OpenVINO ingestion:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab64ffea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!$python main.py -m test export --resume unet_camvid.pth --data data/CamVid -b 1 --to-onnx unet_camvid.onnx"
```

Comment:
I think it may be better to show how to create model and export it to onnx directly in notebook and functions for train test too instead of hide them in script. It may be useful for people who want understand how to quantize own segmentation model trained on own dataset

Reply:
We have other notebooks in this very repository for that.
https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/302-pytorch-quantization-aware-training